packages/react/src/components/ProgressIndicator/ProgressIndicator-story.js

```diff
@@ -16,14 +16,25 @@ import { settings } from 'carbon-components';
 
 const { prefix } = settings;
 
+let spaceEquallyKnob = () => {
+  const spaceEqually = boolean('Space Equally', false);
+  return spaceEqually;
+};
+
+let verticalKnob = () => {
+  const vertical = boolean('Vertical', false);
+  return vertical;
+};
+
 storiesOf('ProgressIndicator', module)
   .addDecorator(withKnobs)
   .add(
     'Default',
     () => (
       <ProgressIndicator
-        vertical={boolean('Vertical', false)}
-        currentIndex={number('Current progress (currentIndex)', 1)}>
+        vertical={verticalKnob()}
+        currentIndex={number('Current progress (currentIndex)', 1)}
+        spaceEqually={spaceEquallyKnob()}>
```

Comment:
Same as above, is there a benefit to this rather than doing `spaceEqually={boolean('Space Equally', false)}>`?